UI/Program.cs

```diff
@@ -30,7 +30,7 @@
     options.Cookie.IsEssential = true;
     options.Cookie.Name = StringConstants.SessionCookieName;
     options.Cookie.HttpOnly = true;
-    options.Cookie.SecurePolicy = CookieSecurePolicy.SameAsRequest;
+    options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
```

Comment:
Will this be an issue on a local developer environment?

Reply:
Not for me - was OK when I ran it in localhost.  Do you want to check it's OK for you?  We already have the compare cookie as secure and seems OK locally, so hopefully not an issue.